cle/backends/backend.py

```diff
@@ -567,6 +564,27 @@ def _checksum(self):
             self.md5 = hashlib.md5(data).digest()
             self.sha256 = hashlib.sha256(data).digest()
 
+    def _create_extra_regions(self) -> None:
```

Comment:
Why is this defined on backend instead of on ihex if it is only called by ihex?

Reply:
It can be implemented by other backends as well if they want to support this feature.